src/color.rs

```diff
@@ -1,3 +1,4 @@
+use ansi_term::Colour;
```

Comment:
There's already a `Color` struct, so to avoid ambiguity, we shouldn't import this, and just refer to it as `ansi_term::Color`. (With the American spelling.)

```suggestion
```